go.mod

```diff
@@ -15,14 +15,17 @@ require (
 	golang.org/x/tools v0.1.6-0.20210820212750-d4cc65f0b2ff
```

Comment:
This scenario does not solve the case: https://github.com/kubernetes-sigs/kubebuilder/pull/2581